docs/head.md

```diff
@@ -57,6 +57,24 @@ export default {
 </script>
 ```
 
+If you wish to access `this` or data on a component/page, make `metaInfo` a function:
+```js
+<script>
+export default {
+  name: 'Post Template',
+  metaInfo() {
+    return {
+      title: this.$context.post.name,
```

Comment:
Can we make this `this.$page` instead? I think it's more common to use the Data API than to generate page data dynamically.

(ditto for line 69)